src/standalone/Soil/Soil.jl

```diff
@@ -157,10 +157,23 @@ function horizontal_components!(
     _...,
 ) end
 
+"""
```

Comment:
These were previously defined in runoff.jl, but are general enough to move here, I thought. We allow users to pass in a tuple of source terms, but in some cases, some boundary conditions require source terms to be added. Using runoff is one example (it affects the BC and the source terms), sublimation (source) + evaporation(boundary flux) is another example. so this is used in the model constructors to be consistent. if the user chooses to use runoff, this will add the correct source term.

https://github.com/CliMA/ClimaLand.jl/blob/1be8238de46bdd233ee838fcdec2e3d35f933680/src/standalone/Soil/runoff.jl#L55